module/gui/Tools.py

```diff
@@ -0,0 +1,518 @@
+# -*- coding: utf-8 -*-
+
+from PyQt4.QtCore import *
+from PyQt4.QtGui import *
+from os.path import join
+from bisect import bisect_left, bisect_right
+from functools import cmp_to_key
+from datetime import datetime
+import logging
+
+def whatsThisFormat(title, text):
+    #html = "<b>" + title + "</b><br><br>" + text
+    html = "<table><tr><td><b>" + title + "</b></td></tr></table>" + "<table><tr><td>" + text + "</td></tr></table>"
+    return html
+
+class WhatsThisButton(QPushButton):
+    def __init__(self, text="?"):
+        QPushButton.__init__(self, text)
+        self.text = text
+        self.setDefault(False)
+        self.setAutoDefault(False)
+        self.setFocusPolicy(Qt.NoFocus)
+        self.updateSize()
+        self.connect(self, SIGNAL("clicked()"), QWhatsThis.enterWhatsThisMode)
+
+    def updateSize(self):
+        width  = self.fontMetrics().boundingRect(self.text).width()
+        height = self.fontMetrics().boundingRect(self.text).height()
+        s = width
+        if height > s:
+            s = height
+        s += 4  # border
+        self.setFixedSize(s, s)
+
+class WtDialogButtonBox(QDialogButtonBox):
+    def __init__(self, orientation=Qt.Horizontal, parent=None):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Qt' may be undefined, or defined from star imports: PyQt4.QtCore, PyQt4.QtGui](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=567903)